src/task/SubscribeAndReceiveSubscribeAckTask.ts

```diff
@@ -107,6 +110,19 @@ export default class SubscribeAndReceiveSubscribeAckTask extends BaseTask {
         }
 
         handleSignalingClientEvent(event: SignalingClientEvent): void {
+          if (event.isConnectionTerminated()) {
+            const message = 'SubscribeAndReceiveSubscribeAckTask connection terminated';
```

Comment:
Should we print more explicit log with the event message and event closeCode like done in the JoinAndReceiveIndexTask here: https://github.com/aws/amazon-chime-sdk-js/blob/master/src/task/JoinAndReceiveIndexTask.ts#L51-L66?

Also, can there be a different statusCode to send below instead of `MeetingSessionStatusCode.SignalingInternalServerError`?

Reply:
Ok, yeah this is fine too.